dev-client/src/screens/ProjectTeamScreen/components/RolePill.tsx

```diff
@@ -19,24 +19,13 @@ import {useTranslation} from 'react-i18next';
 
 import {ProjectMembership} from 'terraso-client-shared/project/projectTypes';
 
-import {Badge} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {Chip} from 'terraso-mobile-client/components/Chip';
 
 type RolePillProps = {
   membership: ProjectMembership;
 };
 
 export const RolePill = ({membership}: RolePillProps) => {
```

Comment:
This can probably be renamed to `RoleChip` and live with the others, it seems potentially more reusable than this screen.